tasks/task341_WinoMT_classification_gender_anti.json

```diff
@@ -0,0 +1,9500 @@
+{
+    "Contributors": [
+        "Xudong Shen"
+    ],
+    "Source": [
+        "WinoMT (dataset: https://github.com/gabrielStanovsky/mt_gender, paper: https://aclanthology.org/P19-1164.pdf)"
+    ],
+    "Categories": [
+        "Classification"
+    ],
+    "Definition": "In this task, you are given a sentence and a profession that is mentioned in the sentence. The gender of the identity with the given profession can be identified using the coreference link with gendered pronouns. You are expected to identify the gender. Select from female and male.",
```

Comment:
I am not quite sure what is the difference between this task and the one defined in `tasks/task340_WinoMT_classification_gender_pro.json`.